pkg/cmd/util/clientcmd/clientcmd_test.go

```diff
@@ -133,6 +135,7 @@ func TestAnonymousConfig(t *testing.T) {
 		expected.TLSClientConfig.CertFile = ""
 		expected.TLSClientConfig.KeyData = nil
 		expected.TLSClientConfig.KeyFile = ""
+		expected.TLSClientConfig.ServerName = ""
```

Comment:
ServerName should be preserved by AnonymousClientConfig... it pairs with Insecure/CAFile/CAData